lib/smart_answer_flows/marriage-abroad/outcomes/_consular_fees_table_items.govspeak.erb

```diff
@@ -1,3 +1,7 @@
+<% if service_counter == 0 %>
```

Comment:
`service_counter` is a bit of Rails magic, see [Layouts and Rendering in Rails > Local Variables](http://guides.rubyonrails.org/layouts_and_rendering.html#local-variables):

> Rails also makes a counter variable available within a partial called by the collection, named after the member of the collection followed by _counter. For example, if you're rendering @products, within the partial you can refer to product_counter to tell you how many times the partial has been rendered. This does not work in conjunction with the as: :value option.

I personally dislike all kind of "magic" but in this case I wouldn't know what else to use instead. This seems to be made for exactly this case.
`calculator.services.any?` wouldn't work as that would show the table header all the time.